src/ServiceControl.Monitoring/Settings.cs

```diff
@@ -54,9 +56,10 @@ public ITransportCustomization LoadTransportCustomization()
         {
             try
             {
-                TransportType = TransportManifestLibrary.Find(TransportType);
-
-                var customizationType = Type.GetType(TransportType, true);
+                var transportManifest = TransportManifestLibrary.Find(TransportType);
+                var assemblyPath = Path.Combine(transportManifest.Location, $"{transportManifest.AssemblyName}.dll");
+                var loadContext = new PluginAssemblyLoadContext(assemblyPath);
```

Comment:
Here we don't need the functions? Would it be worthwhile to still introduce those for similarity reasons?

Reply:
Yep, that was the next step in the changes I wanted to make!